application/src/main/java/org/opentripplanner/transit/model/site/StopLocation.java

```diff
@@ -62,7 +62,7 @@ default String getPlatformCode() {
   }
 
   @Nullable
-  default TransitMode getGtfsVehicleType() {
+  default TransitMode getVehicleType() {
```

Comment:
If you're already renaming this, you could use our naming convention and remove the `get`.